lib/id_manager.c

```diff
@@ -203,6 +222,34 @@ struct mptcpd_idm *mptcpd_idm_create(void)
                 idm = NULL;
         }
 
+        if (_idm_hash_seed == 0) {
+#ifdef HAVE_GETRANDOM
+                /*
+                  Randomize the hash seed value with data from the
+                  kernel random number generator.
+                */
+                ssize_t const len =
+                        getrandom(&_idm_hash_seed,
+                                  sizeof(_idm_hash_seed),
+                                  GRND_NONBLOCK);
+
+                if (len == -1) {  // unlikely
+                        char errmsg[80];
+                        int const r =
+                                strerror_r(errno,
+                                           errmsg,
+                                           L_ARRAY_SIZE(errmsg));
+
```

Comment:
Could also fail over to `l_getrandom_uint32()` here and make it a warning instead of error.

Reply:
Let's just use `l_getrandom_uint32()`.